packages/fxa-payments-server/src/components/SubscriptionForm/index.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { injectStripe, CardElement } from 'react-stripe-elements';
+
+class SubscriptionForm extends React.Component {
+  
+  render() {
+    const {
+      plansByProductId,
+    } = this.props;
+    
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <ul>
+          <li>
+            <p>Plans</p>
+            <ul>
+            {plansByProductId.length === 0 ? (
+              <li>No plans found.</li>
+            ) : (
+              plansByProductId.map(({ plan_id, product_id }) => (
```

Comment:
Should we avoid underscores? I guess I've noticed the pattern of underscores in JSON (googlism?)...

Reply:
We're kind of a mess of camelCase and snake_case in various parts of the stack. This is coming from [the auth-server API which relays from the subhub API](https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/routes/subscriptions.js#L42). We could rename / normalize in the client here or in the auth-server, but subhub probably won't changes. Relaying it direct was the lazy choice.

Reply:
Yes, I agree on not renaming